src/proto/thread_telemetry.proto

```diff
@@ -309,6 +309,25 @@ message TelemetryData {
     optional SrpServerResponseCounters response_counters = 6;
   }
 
+  message TrelPacketCounters {
+    optional uint64 trel_frame_tx = 1;
+    optional uint64 trel_bytes_tx = 2;
+    optional uint64 trel_frame_rx = 3;
+    optional uint64 trel_bytes_rx = 4;
+    optional uint64 trel_frame_tx_failed = 5;
```

Comment:
Should we also use plural forms of `frame` ?

Reply:
Also, what's the difference between `packets` and `bytes`?

Reply:
Using packets now for consistency. A packet contains a sequence of bytes.